doc/ChangeLog

```diff
@@ -0,0 +1,93 @@
+===============================================================
+
```

Comment:
As I was working on a CAM ChangeLog entry, I realized that we need a ChangeLog_template as well, to provide the blank template that we will fill in for future tags.